lib/ain-grpc/src/rpc/eth.rs

```diff
@@ -1229,3 +1234,55 @@ fn sign(
         }
     }
 }
+
+fn error_on_execution_failure(reason: &ExitReason, data: &[u8]) -> RpcResult<()> {
+    match reason {
+        ExitReason::Succeed(_) => Ok(()),
+        ExitReason::Error(err) => {
+            if *err == ExitError::OutOfGas {
+                return Err(internal_err("out of gas"));
+            }
+            Err(internal_err_with_data(format!("evm error: {err:?}"), &[]))
+        }
+        ExitReason::Revert(_) => {
```

Comment:
Revert arm does not contain a field, does not need a placeholder

Reply:
that got something `Revert(ExitRevert)` but here doesn't need it
https://docs.rs/evm-core/latest/src/evm_core/error.rs.html#30